src/test/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/security/JwtServiceTest.kt

```diff
@@ -39,9 +40,20 @@ class JwtServiceTest {
     assertThat(jwtService.authorities(jwt)).containsExactly("ROLE_SLM_CREATE_BARCODE")
   }
 
+  @Test
+  fun `the generated JWT should expire at midnight`() {
+    val jwtService = jwtService()
+    val jwt = jwtService.generateToken("some.email@company.com")
+
+    val expiresAt = jwtService.expiresAt(jwt)
+    assertThat(expiresAt).isEqualTo(midnight())
+  }
+
+  private fun midnight() = Instant.now().plus(1, ChronoUnit.DAYS).truncatedTo(ChronoUnit.DAYS)
```

Comment:
The test code looks suspiciously like the production code. Not sure what else to do?